.circleci/workflows.yml

```diff
@@ -291,6 +294,7 @@ jobs:
     resource_class: small # 1-core
     environment:
       FERROCENE_HOST: x86_64-unknown-linux-gnu
+      FERROCENE_TARGETS: aarch64-unknown-none,aarch64-unknown-linux-gnu,thumbv7em-none-eabi,thumbv7em-none-eabihf,armv8r-none-eabihf,wasm32-unknown-unknown
```

Comment:
I don't like having to duplicate this -- Maybe we can use a yaml alias to avoid the duplication.